client/my-sites/pages/page-list.jsx

```diff
@@ -31,54 +27,47 @@ import {
 	getSelectedSiteId,
 } from 'state/ui/selectors';
 
-var PageList = React.createClass( {
-
-	mixins: [ PureRenderMixin ],
-
-	propTypes: {
-		context: React.PropTypes.object,
-		search: React.PropTypes.string,
-		hasSites: React.PropTypes.bool.isRequired,
-		site: React.PropTypes.object,
-		siteId: React.PropTypes.any
-	},
-
-	render: function() {
-		return (
-			<PostListFetcher
-				type="page"
-				number={ 100 }
-				siteId={ this.props.siteId }
-				status={ mapStatus( this.props.status ) }
-				search={ this.props.search }>
-				<Pages
-					{ ...omit( this.props, 'children' ) }
-				/>
-			</PostListFetcher>
-		);
-	}
-} );
-
-var Pages = React.createClass( {
+const PageList = ( props ) => (
```

Comment:
While we're here, would it make sense to split these components into separate files?